api/app/core/config.py

```
@@ -134,6 +213,61 @@ def FAQ_DB_PATH(self) -> str:
+        if provider not in provider_credentials:
+            raise ValueError(
+                f"Unsupported LLM provider: {provider}. "
+                f"Supported providers: {', '.join(provider_credentials.keys())}"
+            )
+
+        credential = provider_credentials[provider]
+        if not credential and provider != "ollama":
+            raise ValueError(
+                f"{provider.upper()}_API_KEY is required but not set. "
+                f"Please configure the API key for {provider} provider."
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)